### DISCUSSION TITLE
_include/head/custom.html override is not picked up

### DISCUSSION THREAD

**Discussion by Laszlo-JFLMTCO** [mmistakes/minimal-mistakes#3167] [Q&A]
Hi There,

Great theme with tons of potential!
I am trying to use it as remote-theme in github pages.
I added the https://realfavicongenerator.net/ generated code to `_includes/head/custom.html` as per the instructions, but it is never picked up.
Do you have any suggestions what I might be doing wrong?

Thank you.

**Comment by iBug** [comment]
> Do you have any suggestions what I might be doing wrong?

There are so many ways things can go wrong. Without any code (without being able to reproduce) it's impossible to tell where it's wrong. Please share your repository so we can take a look.

**Comment by Campostrini** [comment]
Did you solve it?
